src/ffi/playermove.rs

```diff
@@ -28,11 +49,61 @@ pub const PM_TRACELINE_PHYSENTONLY: TraceLineFlag = 0;
 pub const PM_TRACELINE_ANYVISIBLE: TraceLineFlag = 1;
 pub type TraceLineFlag = c_int;
 
+/* automatically generated by rust-bindgen 0.71.1 */
+
+#[doc = "\tCopyright (c) 1996-2002, Valve LLC. All rights reserved.\n\n\tThis product contains software technology licensed from Id\n\tSoftware, Inc. (\"Id Technology\").  Id Technology (c) 1996 Id Software, Inc.\n\tAll Rights Reserved.\n\n   Use, distribution, and modification of this source code and/or resulting\n   object code is restricted to non-commercial enhancements to products from\n   Valve LLC.  All other use, distribution, or modification is prohibited\n   without written permission from Valve LLC."]
+pub type string_t = c_uint;
+pub type byte = c_uchar;
+pub const qboolean_false_: qboolean = 0;
+pub const qboolean_true_: qboolean = 1;
+pub type qboolean = c_uint;
+#[repr(C)]
+#[derive(Debug)]
```

Comment:
```suggestion
#[derive(Debug, Copy, Clone)]
```
Can remain Clone/Copy because no pointers